front/app/components/CustomFieldsForm/CustomFields.tsx

```diff
@@ -235,8 +235,10 @@ const CustomFields = ({
                 </Text>
               )}
               {renderField({ question, projectId, ideaId })}
-              {(question.key === 'title_multiloc' ||
-                question.key === 'body_multiloc') && <InputIQ phase={phase} />}
+              {question?.code &&
+                ['title_multiloc', 'body_multiloc'].includes(question.code) && (
```

Comment:
Could you create a const with this array. Putting it in the same file is fine, will just make it a bit more readable.